README.md

```diff
@@ -7,6 +7,19 @@ and use of prometheus at scale (when you have more than one prometheus host).
 Promxy delivers this unified access endpoint without requiring **any** sidecars,
 custom-builds, or other changes to your prometheus infrastructure.
 
+## Why promxy?
+[**Detailed version**](https://github.com/jacksontj/promxy/blob/master/STORY.md)
+
+**Short version**:
+Prometheus itself provides no real HA/clustering support. As such best-practice
+is to run multiple hosts with the same config. Similarly prometheus has no real
+built-in query federation, which means that you end up with N sources in grafana
+that is (1) confusing grafana users and (2) disallows aggregation across the sources.
+Promxy supports the HA setup of prometheus by "merging" the data from the duplicate
+hosts (so if there is a gap in one, promxy will fill with the other). In addition
+Promxy provides a single datasource for all promql queries -- meaning your grafana
```

Comment:
"provides a single" I wonder if this is specific to the grafana use case? If so maybe mention it?

"Promxy allows you to globally aggregate [promql](link) queries so that services like [Grafana](link) only need to configure a single datasource reducing operational overhead." ?